ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-release-3.11.yaml

```diff
@@ -17,15 +17,15 @@ images:
 - context_dir: elasticsearch/
   dockerfile_path: Dockerfile.centos7
   from: base
-  to: logging-elasticsearch
+  to: logging-elasticsearch5
```

Comment:
In 3.1??

Reply:
3.11

Reply:
Maybe I misunderstand, but I thought this file was for 3.11 only?